src/validation.cpp

```diff
@@ -4227,6 +4227,14 @@ bool static LoadBlockIndexDB(ChainstateManager& chainman, const CChainParams& ch
     return true;
 }
 
+void CChainState::LoadMempool(const ArgsManager& args)
+{
+    if (args.GetArg("-persistmempool", DEFAULT_PERSIST_MEMPOOL)) {
```

Comment:
nit: could this be passed as a `persistentMempool` bool parameter to try to contain the configuration/command line argument logic in init.cpp ? (Even if there is gArgs logic elsewhere in validation.cpp ..)

Reply:
Actually, module-specific arg parsing and configuration should be moved *out* of init.cpp. For example, the wallet settings have been moved out of init as well.

Reply:
Ok, thanks.